### ISSUE TITLE
Shake triggering just once when using emit_shake_3d

### ISSUE THREAD

**Issue by sourcelocation** [Eneskp3441/Shaker#11]
**Describe the bug**
Camera isnt shaking with an attached component and receiver after first use:
<img width="388" alt="Screenshot 2024-12-28 at 17 22 58" src="https://github.com/user-attachments/assets/04af21c1-975c-4cac-8c23-2304badedfc4" />

**To Reproduce**
1. Create the structure from screenshot
2. Call `Shaker.shake_emit_3d(global_position,preset,10,1)` two times, it will only trigger once

**Desktop (please complete the following information):**
 - OS: macos
 - Shaker Version [e.g. 22] latest

**Comment by sourcelocation** [comment]
Workaround (in comments): https://www.youtube.com/watch?v=Xh0O5odqPOs

**Comment by Eneskp3441** [comment]
Hello! I just tested creating the project from scratch, and I can emit as much as I want.
Make sure that "Auto Play" is set to true in your ShakerComponent and that you've created a "Shaker Preset" preset.

**Comment by sourcelocation** [comment]
Right, but it's a bit counter intuitive having a dummy preset, since you are emitting from the outside.

**Comment by Supamiu** [comment]
This also seems to be happening in 2D, creating a dummy empty preset doesn't fix the issue neither, and even worse: it negates all shaking received.

First shaking works well, after that, it's like nothing gets received at all.

**Comment by Eneskp3441** [comment]
the last time I tried, it worked perfectly without any issues. Could you try the same steps in a new project and send me the project file?

**Comment by Supamiu** [comment]
Sure, I'll try this tomorrow, thanks for the quick response.

**Comment by Supamiu** [comment]
https://github.com/Supamiu/Shaker-Emitter-Repro reproduces the issue the closest I could get to my setup (I'm spawning VFX that have an emitter for shake, then destroying themselves after few seconds).

First click on button works, the others don't.

**Comment by Eneskp3441** [comment]
> Hello! I just tested creating the project from scratch, and I can emit as much as I want. Make sure that "Auto Play" is set to true in your ShakerComponent and that you've created a "Shaker Preset" preset.

Hello, I understand the issue. In the project you shared, the ShakerComponent doesn't have a Shaker Preset. When I created an empty Shaker Preset, it started working multiple times. Normally, it shouldn't work at all. I'm not sure about the cause at the moment, but I will release an update soon that ensures it works without the need to create an empty Shaker Preset. Thank you!
